templates/archives/stories/mayowa_story.jinja2

```diff
@@ -10,13 +10,13 @@ Mayowa Songonuga
 
 
 {% block story_body %}
-<p>Today, Computer Science is the most popular course at MIT. As of 2018, approximately 38.5% of
-    declared students had primary majors in some form of CS (including joint majors such as
-    Computer Science and Molecular Biology). MIT is one of the top institutions in the field, and
-    it’s interesting to think that the transition to a Computer Science powerhouse happened so
-    recently. In fact, it wasn’t until 1975 that the Department of Electrical Engineering was renamed
-    to the Department of Electrical Engineering and Computer Science. That same year, MIT gave out its
-    first undergraduate Computer Science degree.</p>
+    <p>Today, Computer Science is the most popular course at MIT. As of 2018, approximately
+    <a href="https://registrar.mit.edu/stats-reports/majors-count/2017-2018">38.5%</a> of declared students had
+    primary majors in some form of c.s. (including joint majors such as Computer Science and Molecular Biology). MIT is
```

Comment:
I would write out `c.s.` as `Computer Science`, it is more formal and clearer so people who may not know what `c.s.` means can quickly understand.